### ISSUE TITLE
BeagleBoard-xM images don't boot with USB client g_multi loaded

### ISSUE THREAD

**Issue by jadonk** [RobertCNelson/boot-scripts#47]
https://github.com/jadonk/boot-scripts/blob/master/boot/omap3_beagle.sh#L24

Do you know where this file would be enabled in the image builds?

**Comment by RobertCNelson** [comment]
yeah, this needs to be re-worked..  i didn't have g_multi setup on the mainline kernel, and we need to use the common script from am335x_evm.sh
